### ISSUE TITLE
Add isSubscribing flag

### ISSUE THREAD

**Issue by ai** [logux/vuex#10]
In `Vue/Vuex example` we are using data to detect that subscription was loaded:

```js
	<div v-if="counter">
		<h1>{{ counter }}</h1>
		<button @click="increment" />
	</div>
	<div v-else>
		<Loader />
	</div>
```

Unfortunately, it is not a good way for all cases. It is much better to use data from the Logux: when Logux server sent you an initial channel data, it will send you `{ type: 'logux/processed', id: subscribeActionId }`.

In Logux Redux you can use:

```js
store.dispatch.sync({ type: 'logux/subscribe', channel }).then(() => {
  // initial data was loaded
})
```

**Comment by euaaaio** [comment]
@ai, you mean, counter could have been set before Logux sent it's state?

**Comment by ai** [comment]
@euaaaio nope, the problem is different.

1. What is the server’s state of the counter is 0? Or course, we can do something like `if typeof counter !== 'undefined'`. But what if the state of the data is `undefined`?
2. What if we will subscribe to `users` channel? We have 3 users on the server. The server takes 100ms to load each user. So after the subscription, we will have 1st user from the server, but need to wait extra 200ms to load all other users. If we will have `if users` in our template we will render that single user even if the subscription is not finished.

**Comment by euaaaio** [comment]
Released in v0.2.0
